### PR TITLE
Add code

### PR DESCRIPTION
**Pull request by apsavin**
For review @jifeon @AGresvig @frenzzy

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": "0.0.1",
+  "private": true,
+  "name": "logger",
+  "main": "./dist/logger.server.js",
+  "browser": "./dist/logger.client.js",
+  "scripts": {
+    "lint": "LIST=`git diff --cached --name-only --diff-filter=ACMR | grep \"js$\"`; if [ \"$LIST\" ]; then eslint $LIST && jscs $LIST; fi",
+    "build": "babel lib -d dist",
+    "postinstall": "npm run build"
```
Comment (apsavin) [inline package.json:10]: I'm not sure about this line. Maybe, we'll change it in future and will just commit transpiled code.

**Comment by jifeon** [comment]
Looks good!

**Comment by AGresvig** [comment]
Cool!

**Comment by apsavin** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;Nanj6P8O&#x2F;300-employ-improved-logging"> Employ improved logging</a>

**Pull request merged**